static/src/javascripts/projects/commercial/commercial-metrics.ts

```diff
@@ -2,21 +2,20 @@ import type { ABTest } from '@guardian/ab-core';
 import { sendCommercialMetrics } from '@guardian/commercial-core';
 import { getSynchronousTestsToRun } from 'common/modules/experiments/ab';
 import { commercialPartner } from 'common/modules/experiments/tests/commercial-partner';
-import { improveSkins } from 'common/modules/experiments/tests/improve-skins';
-import config_ from '../../lib/config';
+import { improveSkins } from '../common/modules/experiments/tests/improve-skins';
 
-// This is really a hacky workaround ⚠️
-// TODO convert config.js to TypeScript
-const config = config_ as {
-	get: (s: string, d?: unknown) => unknown;
-};
+const { isDev } = window.guardian.config.page;
+const { pageViewId } = window.guardian.ophan;
+const { browserId } = window.guardian.config.ophan;
 
 let logged = false;
-
-const isDev = Boolean(config.get('page.isDev', false));
+const sendMetrics = (): void => {
+	if (logged) return;
+	logged = sendCommercialMetrics(pageViewId, browserId, isDev);
+};
 
 const init = (): Promise<void> => {
-	if (!config.get('switches.commercialMetrics', false))
+	if (!window.guardian.config.switches.commercialMetrics)
 		return Promise.resolve();
 
 	const testsToForceMetricsFor: ABTest[] = [commercialPartner, improveSkins];
```

Comment:
@ashishpuliyel this is how we capture metrics for tests.

Can be tested by adding #ab-ImproveSkins=control